railties/lib/rails/generators/test_unit/system/templates/system_test_helper.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class ActionSystemTestCase < ActionSystemTest::Base
+  teardown do
+    take_failed_screenshot
+    Capybara.reset_sessions!
+  end
+end
```

Comment:
This file is generated when a new app is generated or a scaffold is generated and there is no system_test_helper file.

The reason for this file is there is a ton of other setup that Capybara can do and programmers need a place to put that code. Additionally some may not want to take_failed_screenshots or do other such work. This gives them a clear place to put anything and everything related to system tests or required for setup/teardown on every test.